### DISCUSSION TITLE
Is there a way to tostring() a Document?

### DISCUSSION THREAD

**Discussion by bjuan210302** [abey79/vpype#708] [Q&A] [answered]
With an existing `vpype.Document` is there a way to output its raw SVG as string? I know there's `write_svg` that takes a document, but I'm working from a serverless and can't access a file system.

I there any way of achieving this? The only thing I can think of is straight up copying the `write_svg` and returning `dwg.tostring()` instead of wirting it to a file.

**Comment by abey79** [comment, marked as answer]
`write_svg` takes a `TextIO` as argument. So you can pass a `StringIO`([docs](https://docs.python.org/3/library/io.html#io.StringIO)):

```python
import io
import vpype as vp

string_io = io.StringIO()
doc = vp.Document()
vp.write_svg(string_io, doc)
print(string_io.getvalue())
```

**Reply by bjuan210302** [reply]
Thank you!
